src/versionize/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "versionize"
+version = "0.0.1"
+authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
+
+[dependencies]
+serde = ">=1.0.27"
+serde_derive = ">=1.0.27"
+bincode = "1.2.1"
+versionize_derive = { path = "../versionize_derive" }
+crc64 = "1.0.0"
+vmm-sys-util = "0.4.0"
+
+[dev-dependencies]
+criterion = "0.3.0"
+
+[build-dependencies]
+proc-macro2 = "1.0"
```

Comment:
Actually the question about locking specific versions for dependencies applies to all the new ones. The default so far has been to specify the *minimum* supported version and investigate what, if anything, is broken when dependencies get updated

Reply:
I think we need to discuss more about this topic. As I said earlier, I am not a fan of getting automatic updates just to have the latest version.

Reply:
For versioned serialization is tricky to do the updates. I recommend to use `=1.0` instead. This way we have more control on the updates. I believe we do need to make sure that we do not fall behind with crates versions because newer versions of crates come with bug fixes, performance improvements, new features that we might need. They might also come with breaking changes, and in the versioned serialization case we need to be extra careful about it.